uint8 num
uint8[] id
---
bool success
